internal/testutil/acc/pre_check.go

```diff
@@ -18,6 +18,16 @@ func PreCheckBasic(tb testing.TB) {
 	}
 }
 
+func SkipIfTPFAdvancedCluster(tb testing.TB, existingPreCheck func()) func() {
+	tb.Helper()
+	return func() {
+		if os.Getenv("MONGODB_ATLAS_TPF_ADV_CLUSTER_TESTS") == "true" {
+			tb.Skip("Skipping TPF Advanced Cluster tests")
```

Comment:
nit: My understanding is not that we are skipping TPF advanced cluster tests but that we don't run tests on TPF advanced clusters at all. Is this a correct assumption? If so, I think a message more along the lines of "Skipping tests as resource is TPF Advanced Cluster" would make this more explicit.

Reply:
thank you! Updated in 053deb331cb7c3189a7de3ffa47e597ba5721ac1, WDYT?

Reply:
Please Tag/DM if you want an update otherwise I might not see it 😅